src/diffpy/structure/lattice.py

```diff
@@ -148,13 +228,29 @@ def setLatPar(self, a=None, b=None, c=None,
             alpha=None, beta=None, gamma=None, baserot=None):
         """set lattice parameters and all related tensors
 
-        a, b, c, alpha, beta, gamma -- lattice parameters, unit cell angles
-                    are in degrees.
-        baserot  -- unit cell rotation, base = stdbase @ baserot
-
-        Note: parameters with value None will remain unchanged.
-
-        No return value.
+        Parameters
+        ----------
+        a : float, optional
+            Lattice constant of a-axis, default to None.
```

Comment:
Drop *default to None*.  Here `None` means not specified, i.e., the associated lattice parameter is thus left unchanged.

```
a : float, optional
    New value for the cell length `a`.